apps/dokuha-roadmap-frontend/src/const/user.ts

```diff
@@ -0,0 +1 @@
+export const USER_ID = process.env.NEXT_PUBLIC_LOCAL_USER_ID;
```

Comment:
[nits]
一応こっちもデフォルト値を入れておこう
```suggestion
export const USER_ID = process.env.NEXT_PUBLIC_LOCAL_USER_ID ?? '00000000-0000-0000-0000-000000000001';
```